homework_1/zad3/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/sethgrid/pester"
+	"io/ioutil"
+	"log"
+	"strings"
+)
+
+type pokemonInformation struct {
+	Name string
+	LocationAreaEncounters string `json:"location_area_encounters"`
+}
+
+type pokemonLocation struct {
+	LocationArea locationArea `json:"location_area"`
+}
+
+type locationArea struct {
+	Name string
+}
+
+const pokemonURL= "https://pokeapi.co/api/v2/pokemon/"
```

Comment:
API parsing specific things should not be within the main package - please extract them to a separate package